### ISSUE TITLE
no display data. no buffer in display.

### ISSUE THREAD

**Issue by feipinxiang** [zappar-xr/universal-ar-unity#22]
**Describe the bug**
[zcv::Log][2023-06-19 11:54:24] Start
[zcv::Log][2023-06-19 11:54:24] D3D11 graphics device loaded
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO pipeline_t initialized
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO camera_source_t initialized
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO image_tracker_t initialized
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO loading target from memory: 335036 bytes
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO image target loaded
**[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO no buffer in display
[zcv::Log][2023-06-19 11:55:08] [Zappar] INFO no display data**

**To Reproduce**
1. Used UAR version '3.2.2'
2. With Unity version '2021.3.20f1/2020.3.46f1' using Built-in Render Pipeline and build platform as windows

step:
1.import git package
2.import examples project.
3.open ImageTracker scene.
4.push play button in editor and get the error message

**Desktop (please complete the following information):**
 - OS:Windows 10-22H2

Project And Log file:

[ZuanTouAR.zip](https://github.com/zappar-xr/universal-ar-unity/files/11797096/ZuanTouAR.zip)

**Comment by feipinxiang** [comment]
I just noticed that it doesn't support building for the Windows platform. I switched to android platform and the error still occurs.

**Comment by nfynt-zap** [comment]
Hello @feipinxiang,

The `no buffer in display` and `no display data` points to issue with accessing camera frames. Can you check if your in-built or attached camera is enabled and accessible? From editor you can check whether ZapparCamera component has any dropdown list of active camera devices or not.

Also, you should be able to do In-Editor testing with both Windows and Android platform, it's only the standalone build (Win or Mac) that we don't support. Still switching to target platform is recommended, you can read a bit more about platform support [here](https://github.com/zappar-xr/universal-ar-unity#platform-support).

**Comment by feipinxiang** [comment]
The webcamera can be recognized. 

> Hello @feipinxiang,
> 
> The `no buffer in display` and `no display data` points to issue with accessing camera frames. Can you check if your in-built or attached camera is enabled and accessible? From editor you can check whether ZapparCamera component has any dropdown list of active camera devices or not.
> 
> Also, you should be able to do In-Editor testing with both Windows and Android platform, it's only the standalone build (Win or Mac) that we don't support. Still switching to target platform is recommended, you can read a bit more about platform support [here](https://github.com/zappar-xr/universal-ar-unity#platform-support).


![image](https://github.com/zappar-xr/universal-ar-unity/assets/2398548/c264c7df-6dbc-4408-bc00-a749d3a91d1b)

**Comment by nfynt-zap** [comment]
Does this name of EditorCamera matches with the name shown in your device manager app. I am suspecting the `?` symbol should be `©` here.

<img src="https://github.com/zappar-xr/universal-ar-unity/assets/84909983/7098a714-c404-467f-8466-a6bea8570363" width=50%/>

It's likely the MS Media Foundation encountered some issues while enumerating supported video formats or UVC class for this webcam. You can try upgrading device driver if you think it might help or you may try editing the EditorCamera property from editor by running it in `Debug mode`.

<img src="https://github.com/zappar-xr/universal-ar-unity/assets/84909983/589d35fb-d369-48db-bafc-c480c8b67bcc" width=50% />

Note that this shouldn't effect your build... you can still build it for Android and test. Or try another webcam in-editor if you have one handy. I'll add this to my list of things to do and will update this thread if  I come across anything more.

**Comment by feipinxiang** [comment]
Yes. The full name of the camera is "Microsoft® LifeCam Studio(TM)". I switched to another new camera and the camera image can be displayed normally.
![image](https://github.com/zappar-xr/universal-ar-unity/assets/2398548/422d7cff-e095-4ab4-b066-db50039bdf4d)

**Comment by giganticlab** [comment]
USB2.0 HD UVC WebCam not work

**Comment by rcstamp** [comment]
Still an issue. Imported a blank project and the Zapper SDK, my webcam appears (Brio Logitech 4K) but the feed is black

**Comment by Archi97** [comment]
@rcstamp  any luck fixing this?
